tests/result.txt

```diff
@@ -0,0 +1,15 @@
+============================= test session starts ==============================
+platform linux2 -- Python 2.7.11, pytest-2.9.2, py-1.4.31, pluggy-0.3.1 -- /home/david/miniconda2/bin/python
+cachedir: ../.cache
+rootdir: /home/david/pydexter, inifile: 
+collecting ... collected 7 items
+
+unit.py::test_hist PASSED
+unit.py::test_scatter_with_snd_arg PASSED
+unit.py::test_scatter_without_snd_arg PASSED
+unit.py::test_plot PASSED
+unit.py::test_pie PASSED
+unit.py::test_timeline PASSED
+unit.py::test_bar PASSED
+
+=========================== 7 passed in 0.09 seconds ===========================
```

Comment:
This is generated by the testing library you're using right? This file should be added to the `.gitignore`.